homeassistant/components/technicolor/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "technicolor",
+  "name": "Technicolor",
+  "documentation": "https://www.home-assistant.io/integrations/technicolor",
+  "requirements": ["pytechnicolor==1.0.2"],
```

Comment:
This package uses webscraping to gather information, unfortunately, this is not allowed for our core integrations.

Example upstream code that scrapes: <https://github.com/shaiu/techicolorgateway/blob/96f5ca36cff573280fc50c4840d9d4b0c9b56960/technicolorgateway/modal.py#L17-L21>



For more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0004-webscraping.md>

Reply:
bummer.
Thanks anyway